libdirections-offboard/src/main/java/com/mapbox/navigation/route/offboard/router/NavigationWalkingOptions.kt

```diff
@@ -0,0 +1,76 @@
+package com.mapbox.navigation.route.offboard.router
+
+import com.mapbox.api.directions.v5.WalkingOptions
+
+/**
+ * Class for specifying options for use with the walking profile.
+ */
+class NavigationWalkingOptions internal constructor(val walkingOptions: WalkingOptions) {
```

Comment:
Look at comment above 👀 

Right now we're not taking `Route` / `RouteOptionsNavigation` (`NavigationRoute`) as a parameter so none of the options from `NavigationRoute` can be configured from a client point of view. Should we request a `Route` / `RouteOptionsNavigation` (`NavigationRoute`) object here (that includes `origin`, `waypoints`, `destination` + other fields) instead of `origin`, `waypoints` and `destination`? That was the purpose of having `NavigationRoute` and we're not using it, we're only setting these 3 parameters and not considering the rest.

Currently the builder for configuring the different options live in `WalkingOptionsNavigation` (`base` module).